src/FsAutoComplete/FsAutoComplete.Suave.fs

```diff
@@ -136,7 +136,8 @@ let start (commands: Commands) (args: ParseResults<Options.CLIArguments>) =
                     let res = commands.CompilerLocation() |> List.toArray |> Json.toJson
                     return! Response.response HttpCode.HTTP_200 res httpCtx
                 }
-            path "/lint" >=> handler (fun (data: LintRequest) -> commands.Lint data.FileName)
+            path "/lint" >=> handler (fun (data: FileRequest) -> commands.Lint data.FileName)
+            path "/unusedDeclarations" >=> handler (fun (data: FileRequest) -> commands.GetUnusedDeclarations data.FileName)
```

Comment:
adding things like this, as individual stuff, api become a chatty.
maybe add a `/quickfixes` who return an array of stuff?
i know there is cache, but things like `checker.TryGetRecentCheckResultsForFile` can be done just once, and cancel the request may be easier later

Reply:
I think it's better to have separate endpoints so we can progressively add diagnostics on editor side instead of waiting for all analyzers to finish before showing anything. Especially if some of them can be slow/expensive (for example simplify name according to our Slack discussion). Cancellation is not a problem with our new cancellation system- it will happen automatically

Reply:
fine for me :D